channels-rke2.yaml

```diff
@@ -1870,3 +1870,44 @@ releases:
         repo: rancher-rke2-charts
         version: v4.0.2-build2024020802
     featureVersions: *featureVersions-v1
+  - version: v1.27.13-rc4+rke2r1
+    minChannelServerVersion: v2.7.11-alpha1
+    maxChannelServerVersion: v2.8.99
```

Comment:
curious/double checking.. should this be v2.7.99 for for dev-v2.7 branch?

Reply:
Should be fine either way!